src/renderer/components/ContextMenu.tsx

```diff
@@ -0,0 +1,319 @@
+import React, { useState, useEffect, useLayoutEffect, useContext } from 'react'
+import { useKeyBindingAction, KeybindAction } from '../keybindings'
+import { useRef } from 'react'
+import { ScreenContext, unwrapContext } from '../contexts'
+
+export type ContextMenuItem = { label: string; action: () => void }
+
+type showFnArguments = {
+  cursorX: number
+  cursorY: number
+  items: ContextMenuItem[]
```

Comment:
I have the feeling that this could be a problem at some point to not be able to render own html in the ContextMenu. I guess it's easy to change but we should at least introduce a tr/divider. Besides that really like it to not need to define the items with jsx :)

Reply:
ContextMenuItem type could be a type that can be different types, like
```ts
type ContextMenuItem = {label:string, action:Function} | { divider:true }
```
But as this needs also be taken into account in the estimation how tall the dialog will probably be and the menu has other requirements (needs to open in a static place) I'm not sure if its worth to reuse the context menu instead of doing something extra for the menu.